README.md

```diff
@@ -79,24 +79,23 @@ To maintain a pristine source tree, CMake encourages to perform an out-of-source
 
 ### Building on POSIX systems
 
-    $ mkdir build && cd build
-    $ cmake ..
-    $ cmake --build .
-    $ ctest  # run the test suite
-    $ sudo cmake --install .  # optional
+    $ cmake -B build
+    $ cmake --build build
+    $ ctest --test-dir build  # run the test suite
+    $ sudo cmake --install build  # optional
```

Comment:
If you touch this anyway, perhaps it could be made clear that `build` is a directory here (and it can't hurt to  explain the steps in more detail).

```suggestion
    $ cmake -B build  # Generate a build system in subdirectory "build"
    $ cmake --build build  # Run the actual build process
    $ ctest --test-dir build  # Run the test suite
    $ sudo cmake --install build  # Install the library into the system (optional)
```

Reply:
Sounds good, taken as suggested!